src/PaymentGateways/PayPalCommerce/Repositories/PayPalOrder.php

```diff
@@ -94,7 +94,7 @@ public function createOrder( $array ) {
 			'purchase_units'      => [
 				[
 					'reference_id'        => get_post_field( 'post_name', $array['formId'] ),
-					'description'         => '',
+					'description'         => $array['formTitle'],
```

Comment:
@ravinderk have you checked the max length this can be in PayPal and verified that donation forms with really long names don't cause the request to fail and are properly truncated?

Reply:
@DevinWalker you are really fast on this 👍 . I was testing my change for the character limit.

Reply:
fixed: https://github.com/impress-org/givewp/compare/f56cb66a7ea2...4d372e91cff2